### ISSUE TITLE
Experiments not happening since upgrade

### ISSUE THREAD

**Issue by jonathanmetzman** [google/fuzzbench#1579]
Reported here: https://github.com/google/fuzzbench/pull/1571#issuecomment-1340855489 and here https://github.com/google/fuzzbench/pull/1573
partial fix: https://github.com/google/fuzzbench/pull/1574

**Comment by jonathanmetzman** [comment]
partial fix: https://github.com/google/fuzzbench/pull/1578

**Comment by jonathanmetzman** [comment]
CC @Alan32Liu

**Comment by oliverchang** [comment]
is this still a problem?

**Comment by jonathanmetzman** [comment]
There's another issue that you (partially?) fixed here: https://github.com/google/fuzzbench/pull/1581
By making all of the bug-based benchmarks into coverage benchmarks they were getting built in coverage experiments.

**Comment by jonathanmetzman** [comment]
What was the rationale for doing this btw?

**Comment by jonathanmetzman** [comment]
I need to make these back into bug-based:
diff --git a/benchmarks/bloaty_fuzz_target/Dockerfile b/benchmarks/bloaty_fuzz_target/Dockerfile
diff --git a/benchmarks/libarchive_libarchive_fuzzer/build.sh b/benchmarks/libarchive_libarchive_fuzzer/build.sh
diff --git a/benchmarks/libhtp_fuzz_htp/Dockerfile b/benchmarks/libhtp_fuzz_htp/Dockerfile
diff --git a/benchmarks/libjpeg-turbo-07-2017/Dockerfile b/benchmarks/libjpeg-turbo-07-2017/Dockerfile
diff --git a/benchmarks/libxml2_libxml2_xml_reader_for_file_fuzzer/build.sh b/benchmarks/libxml2_libxml2_xml_reader_for_file_fuzzer/build.sh
diff --git a/benchmarks/mbedtls_fuzz_dtlsclient/Dockerfile b/benchmarks/mbedtls_fuzz_dtlsclient/Dockerfile
diff --git a/benchmarks/mruby-2018-05-23/build.sh b/benchmarks/mruby-2018-05-23/build.sh
--- a/benchmarks/mruby-2018-05-23/build.sh
diff --git a/benchmarks/njs_njs_process_script_fuzzer/Dockerfile b/benchmarks/njs_njs_process_script_fuzzer/Dockerfile
diff --git a/benchmarks/njs_njs_process_script_fuzzer/build.sh b/benchmarks/njs_njs_process_script_fuzzer/build.sh
diff --git a/benchmarks/openh264_decoder_fuzzer/build.sh b/benchmarks/openh264_decoder_fuzzer/build.sh
diff --git a/benchmarks/php_php-fuzz-execute/build.sh b/benchmarks/php_php-fuzz-execute/build.sh
diff --git a/benchmarks/php_php-fuzz-parser-2020-07-25/build.sh b/benchmarks/php_php-fuzz-parser-2020-07-25/build.sh
diff --git a/benchmarks/proj4_standard_fuzzer/build.sh b/benchmarks/proj4_standard_fuzzer/build.sh
diff --git a/benchmarks/re2-2014-12-09/Dockerfile b/benchmarks/re2-2014-12-09/Dockerfile
--- a/benchmarks/re2-2014-12-09/Dockerfile
+++ b/benchmarks/re2-2014-12-09/Dockerfile
grep: (standard input): binary file matches
diff --git a/benchmarks/stb_stbi_read_fuzzer/build.sh b/benchmarks/stb_stbi_read_fuzzer/build.sh
diff --git a/benchmarks/vorbis-2017-12-11/Dockerfile b/benchmarks/vorbis-2017-12-11/Dockerfile
diff --git a/benchmarks/wireshark_fuzzshark_ip/build.sh b/benchmarks/wireshark_fuzzshark_ip/build.sh

**Comment by jonathanmetzman** [comment]
> is this still a problem?

It was a problem as of my last comment. But it may be fixed by https://github.com/google/fuzzbench/pull/1586

**Comment by Alan32Liu** [comment]
I am pretty sure this has been fixed after #1587 and #1588, as I had several coverage-based and bug-based experiments.
Hope you would not mind me closing this for now : )
Please feel free to reopen this and ping me if you noticed any undiscovered errors later.
